lib/active_shipping/carriers/fedex.rb

```diff
@@ -137,7 +137,11 @@ class FedEx < Carrier
     DEFAULT_LABEL_STOCK_TYPE = 'PAPER_7X4.75'
 
     # Available return formats for image data when creating labels
-    LABEL_FORMATS = ['DPL', 'EPL2', 'PDF', 'ZPLII', 'PNG'] 
+    LABEL_FORMATS = %w(DPL EPL2 PDF ZPLII PNG).freeze
+
+    TRANSIENT_TRACK_RESPONSE_CODES = %w(9035 9040 9041 9045 9050 9055 9060 9065 9070 9075 9085 9086 9090).freeze
+
+    UNRECOVERABLE_TRACK_RESPONSE_CODES = %w(9080 9081 9082 9095 9100).freeze
```

Comment:
Yeah, at first I was thinking of doing three patterns, a `TRANSIENT`/`UNRECOVERABLE`/`UNKNOWN` but then I realized it doesn't really make sense to separate known unrecoverable with unknowns because it's really boils down to whether it's transient or not. 

Still good to have the constants though, in case in the future we want to discern between known unrecoverable failures and unknown unrecoverable errors.